application/templates/static_site/static_pages/ethnicity_in_the_uk/ethnic_groups_and_data_collected.html

```diff
@@ -1,95 +1,257 @@
+
+        <p class="govuk-body">A single, standardised list makes it easier to analyse ethnicity data wherever it is collected.</p>
+
+        <p class="govuk-body">However, in practice, not all departments and organisations use the standardised list. For example, because:</p>
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>they need data on some ethnic groups, but not others</li>
+          <li>they want their data to be consistent with data they’ve collected previously, which uses different ethnic groups</li>
+          <li>the low number of people surveyed in some ethnic groups means data has to be combined to make it more reliable, or to protect individual identities</li>
+        </ul>
+
+        <p class="govuk-body">The <a href="https://www.ethnicity-facts-figures.service.gov.uk/dashboards/ethnicity-classifications">dashboard of ethnic categories</a> shows that 20 different ethnicity classifications are used on Ethnicity facts and figures.</p>
+
+      {% endset %}
+
+      {% set scotland_and_northern_ireland_html %}
+
+        <p class="govuk-body">The Censuses in Scotland and Northern Ireland have differences in the ordering of the ethnic groups as well as the groups themselves.
+
+        <h3 class="govuk-heading-s">Northern Ireland</h3>
+
+        <p class="govuk-body">The ethnicity question in the <a href="https://www.nisra.gov.uk/publications/2011-census-questionnaires">2011 Census in Northern Ireland</a> asked respondents to choose from the following 11 ethnic groups:</p>
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>White</li>
+          <li>Chinese</li>
+          <li>Irish Traveller</li>
+          <li>Indian</li>
+          <li>Pakistani</li>
+          <li>Bangladeshi</li>
+          <li>Black Caribbean</li>
+          <li>Black African</li>
+          <li>Black Other</li>
+          <li>Mixed ethnic group (write in)</li>
+          <li>Any other ethnic group (write in)</li>
+        </ul>
+
+        <p class="govuk-body">The <a href="https://gss.civilservice.gov.uk/guidances/harmonisation/0-harmonised-principles/ethnicity/#northern-ireland-input-">GSS harmonised principle for ethnicity in Northern Ireland</a> proposes almost the same standardised ethnic categories as the 2011 Census of England and Wales. However, there are 2 White ethnic groups instead of 4:</p>
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>White</li>
+          <li>Irish Traveller</li>
+        </ul>
+
+        <h3 class="govuk-heading-s">Scotland</h3>
+
+        <p class="govuk-body">The <a href="https://www.scotlandscensus.gov.uk/variables-classification/ethnic-group">ethnicity question in the 2011 Census in Scotland</a> asked respondents to choose from the following 19 ethnic groups:</p>
+
+        <p class="govuk-body">White</p>
```

Comment:
Feels like these should be bold, the same as broad ethnic groups listed in the top paragraph.  I appreciate that then they will look like the same level heading as "Scotland" above though.

Reply:
Yeah, we ran out of heading levels... Nadia thought it was ok as-is, and I think re-using the same bold style as the heading would be confusing.